postgres-appliance/configure_spilo.py

```diff
@@ -106,7 +113,9 @@ def deep_update(a, b):
       parameters:
         archive_mode: "on"
         archive_timeout: 1800s
+        {{#USE_WALE}}
```

Comment:
If we have:
```yaml
archive_mode: "on"
archive_command: 
```

Then xlog will accumulate. We can also set 
```yaml
archive_mode: "off"
```
if we don't have an actual archive_command.